rhel7/fixes/bash/root_bashrc_not_modified.sh

```diff
@@ -0,0 +1,4 @@
+# platform = Red Hat Enterprise Linux 7
+
+rhel7_bashrc_base64_encoded="IyAuYmFzaHJjCgojIFVzZXIgc3BlY2lmaWMgYWxpYXNlcyBhbmQgZnVuY3Rpb25zCgphbGlhcyBybT0ncm0gLWknCmFsaWFzIGNwPSdjcCAtaScKYWxpYXMgbXY9J212IC1pJwoKIyBTb3VyY2UgZ2xvYmFsIGRlZmluaXRpb25zCmlmIFsgLWYgL2V0Yy9iYXNocmMgXTsgdGhlbgoJLiAvZXRjL2Jhc2hyYwpmaQo="
```

Comment:
@matejak In general, this is fine and I understand the use of `base64` when wishing to handle binary data, but `/root/.bashrc` shouldn't contain arbitrary binary data, and more importantly, this one from `rhel7` doesn't. 

I'd personally prefer the form of:

```bash
cat << '_EOF' > /root/.bashrc
# .bashrc

# User specific aliases and functions

alias rm='rm -i'
alias cp='cp -i'
alias mv='mv -i'

# Source global definitions
if [ -f /etc/bashrc ]; then
	. /etc/bashrc
fi
_EOF
```

This way, it is clear to users what the blob is that they're writing to root's `~root/.bashrc`. :)